benchmarks/sum/titan_no_checks.c

```diff
@@ -0,0 +1,127 @@
+/* This file was generated by the Titan compiler. Do not edit by hand */
```

Comment:
The titan compiler is still evolving very fast. I think it might be a good idea to avoid committing these "artisanal" .c files for now.

Reply:
Agreed